com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/transaction/ZigBeeTransactionQueue.java

```diff
@@ -124,6 +124,18 @@ protected ZigBeeTransactionQueue(String queueName, IeeeAddress deviceIeeeAddress
         this.deviceIeeeAdress = deviceIeeeAddress;
     }
 
+    /**
+     * Constructs a {@link ZigBeeTransactionQueue}
+     *
+     * @param queueName a queue name - used for logging to differentiate multiple queues
+     * @param deviceIeeeAddress - the {@link IEEEAddress} of the device for which this queue holds transactions
+     * @param networkManagerId - the {@link IEEEAddress} of the gateway
+     */
+    protected ZigBeeTransactionQueue(String queueName, IeeeAddress deviceIeeeAddress, String networkManagerId) {
```

Comment:
Je ne pense pas que ce soit une bonne idee. L'auteur de la librairie a deja prevu un moyen de customiser le logging pour identifier la queue. Donc, j'aurais utilise ce mecanisme en passant un queue name different (comme `networkManager.getNetworkManagerId() + " - " + node.getIeeeAddress().toString()`).

Par contre, il faut s'assurer que `this.queueName` n'est utilise seulement pour du logging et rien de plus, sinon on risque des effets de bord.

Reply:
Bien vu merci.